Services/Init/classes/class.ilErrorHandling.php

```diff
@@ -415,20 +422,17 @@ public function handlePreWhoops(int $level, string $message, string $file, int $
                 if ($level >= E_USER_NOTICE) {
                     if ($ilLog) {
                         $severity = Whoops\Util\Misc::translateErrorCode($level);
-                        $ilLog->write("\n\n" . $severity . " - " . $message . "\n" . $file . " - line " . $line . "\n");
+                        $ilLog->write("\n\n" . $severity . ' - ' . $message . "\n" . $file . ' - line ' . $line . "\n");
                     }
                     return true;
                 }
             }
 
-            // trigger whoops error handling
             if ($this->whoops instanceof RunInterface) {
                 return $this->whoops->handleError($level, $message, $file, $line);
             }
-            if ($this->whoops) {
-                return $this->whoops->handleError($level, $message, $file, $line);
-            }
```

Comment:
`$this->whoops` is either `NULL` or an instance of the `RunInterface`, so the second `if` condition is redundant.